src/services/index.ts

```diff
@@ -0,0 +1,16 @@
+import {isString, isUndefined} from 'lodash';
+import XDate from 'xdate';
+const {parseDate, toMarkingFormat} = require('../interface');
+
+export function getCalendarDateString(date?: any) {
```

Comment:
Why are you using `any` for the date type?

Reply:
You can pass different date formats and I'll parse it and return the date string in the required format. For example, you can pass Date(), number timestamp and DateData object, like {timestamp: xxxx} or {year: xxxx, month: xx, day: xx}

Reply:
I personally think you should support a single format/type.
No reason to make it difficult for us, the user should know what to pass and should handle on their end. 

Anyway, if you do decide to support multiple formats, I suggest having a union type and not `any`. 
for example: `date?: number | string | Date` 

In any case I would avoid the dateData object cause it's not a trivial date type and maybe we'll end up deprecating it during our refactor, so better not have it here.

Reply:
If I support only one format then I don't see a point in this tool - The user can pass the required string format and that's it. I can limit the support to the types you mention as they make the most sense. The support for DateData was for values returned from Calendars, as there are few callbacks that return this type and I thought it will benefit the user that it doesn't need to parse or extract from it in order to resent a value to Calendars, but it's not a must...

Reply:
Done